docs/books/admin_guide/03-commands.md

```diff
@@ -1596,7 +1596,7 @@ none        on  /proc/sys/fs/binfmt_misc                   type  binfmt_misc  (r
 
 The `;` character strings the commands.
 
-The commands will all run sequentially in the order of input once the user presses <kbd>ENTER</kbd>.
+The commands will all run sequentially in the order of input once the user presses ++ENTER++.
```

Comment:
```suggestion
The commands will all run sequentially in the order of input once the user presses ++enter++.
```

Reply:
@nishaaaaaant Aren't you glad you only started with two files? ;-)... You say it is ready for merging, but as noted by @SergeCroise comment above (and looking down through the document) it doesn't look like it is complete yet. I know it is a lot to look at, which is why taking things in small bites for fixes like this is always best. As already stated elsewhere, keyboard commands are always lower case (i.e. ++enter++) unless placed in quotes. In this case quotes would not be appropriate.